src/elements/fields/PinInput.jsx

```diff
@@ -33,6 +34,9 @@ function SingleOtpInput({
             inputEl.focus();
             inputEl.select();
         }
+        if (paste) {
```

Comment:
should this be wrapped in the previous if statement

Reply:
also can this behavior be replicated by just not calling inputEl.select()?

Reply:
Putting it inside the if makes sense to me, but select and focus both seem to do the same thing, and not calling either of them without doing the selection space thing I'm doing makes it very unclear where your cursor is after pasting